modules/otel-config-instrumentation.adoc

```diff
@@ -98,6 +98,105 @@ spec:
 
 |===
 
+[id="otel-instrumentation-options_exporter_{context}"]
+== Exporter configuration
+
+Although the `Instrumentation` custom resource supports setting up one or more exporters per signal, auto-instrumentation configures only the OTLP Exporter. So you must configure the endpoint to point to the OTLP Receiver on the Collector.
+
+.Sample exporter TLS CA configuration using a config map
+[source,yaml]
+----
+# ...
```

Comment:
Should we have the type/kind headers in these examples? Not sure if it applies in this example, but wanted to call it out.

Reply:
Thank you for the suggestion.
I wanted to keep these YAML as show as possible, but having the kind header makes it easier to read.
I've added the headers.